doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+# z390 Options
+
+The scope value details where the option is used.
+
+* M - mz390 macro processor which reads ASCII assembler source and generates expanded BAL
+* A - az390 assembler which reads BAL and generated OBJ relocatable object code
+* L - lz390 linker which reads OBJ relocatable object files and generates 390 load module
+* E - ez390 emulator runtime which loads and executes 390 load module
+
+!!! Note "Use of parameters with parentheses in *nix shells"
+    Unix shells like sh and bash interpret parentheses if not quoted. For this reason, if you need to specify a z390 parameter that contains
+    parentheses, then that parameter should be quoted. For example, the SYSMAC parameter is quoted as it uses parentheses, whereas the ALIGN parameter is not.
+
+        "SYSMAC(foldername)" ALIGN 
+
+Option | Scope | Default | Description  
+-------|-------|---------|--------------
+@filename	| MALE	| Options in Z390.OPT file if found. |	Retrieve additional options from free form text file with default suffix OPT.  Options can be specified delimited by spaces on as many lines as requires.  All characters on a line following * are ignored as comments. The @file option can be nested.  The default path is the program path.  If a file named Z390.OPT exists in the z390 install directory, these options will be applied first as default options.
```

Comment:
Line 18. Second sentence. "... lines as requires." -> "... lines as required.".

Reply:
Fixed in dcfb0dc